doc/_ext/collapsible/collapsible.py

```diff
@@ -0,0 +1,70 @@
+import os
+
+from docutils import nodes
+from docutils.parsers.rst import Directive
+from docutils.parsers.rst.directives import unchanged
+from sphinx.util.fileutil import copy_asset
+
+collapsible_id_counter = 0
+
+
+class CollapsibleNode(nodes.General, nodes.Element):
+    pass
+
+
+def visit_collapsible_node(self, node):
+    self.body.append(f"""
+<input type="checkbox" id="{node["id"]}" class="hide">
+<label for="{node["id"]}"><span class="collapser">&#x276f;</span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;{node["header"]}</label>
+<div>
+""")
+
+
+def depart_collapsible_node(self, node):
+    self.body.append("""
+</div>
+""")
+
+
+def visit_container(self, node):  # Not displaying headers, unfortunately
+    self.visit_container(node)
+
+
+def depart_container(self, node):
+    self.depart_container(node)
+
+
+class CollapsibleDirective(Directive):
+    has_content = True
+    option_spec = {'header': unchanged}
+
+    def run(self):
+        global collapsible_id_counter
+        collapsible_node = CollapsibleNode('\n'.join(self.content),
+                                           id="collapsible-" + str(collapsible_id_counter),
+                                           header=self.options.get("header", "Click here to show/hide"))
+        collapsible_id_counter += 1
+        self.state.nested_parse(self.content, self.content_offset, collapsible_node)
+
+        return [collapsible_node]
+
+
+def add_css_overrides(app):
+    copy_asset(os.path.join(os.path.dirname(__file__), 'css'),
+               os.path.join(app.builder.outdir, '_static'))
```

Comment:
Could you, please, check that target directory is always "_static"?
My guess is that there is a specific property inside builder object for path to static assets.

Reply:
I can't see it pointed out directly, but in Sphinx API documentation the '_static' directory is hardcoded to their example and is not in any way configurable:
https://www.sphinx-doc.org/en/master/extdev/appapi.html#sphinx.application.Sphinx.add_css_file

Reply:
Looking into sources of sphinx html builder, I can confirm that "_static" is hardcoded there multiple times.